apps/server/src/config/connection.js

```diff
@@ -1,7 +1,8 @@
 const { connect, connection } = require('mongoose');
+require('dotenv').config()
 
 const connectionString =
-  process.env.MONGODB_URI || 'mongodb://localhost:27017/pokerRoll';
+  process.env.MONGODB_URI || process.env.DATABASE_CONNECTION;
```

Comment:
Nice, you can get rid of `MONGODB_URI` since you're calling this `DATABASE_CONNECTION`. When we deploy somewhere we'll have to add an environment variable on the platform with the same name.

Reply:
done